database.c

```diff
@@ -967,3 +971,22 @@ int database_consumer(replay_item *item) {
 	debug(3, "Leaving database_consumer%s\n", "");
 	return rc;
 }
+
+int database_consumer_dry_run(replay_item *item) {
+	const replay_type type = replay_get_type(item);
```

Comment:
I also think that we should have more statistics in the dry-run case:

- `stat_actions` (number of calls to the server)
- `stat_sesscnt` (total number of sessions)
- `stat_sessmax` (maximum number of concurrent sessions)

Reply:
I'll look into that

Reply:
Seeing that there is no server involved, what value should be ```stat_actions``` ?
For ```stat_sesscnt``` and ```stat_sessmax```, as I don't deal with sessions, I'll have to dig further in the non dry run code to see how the values are computed

Reply:
`stat_actions` is just the total number of all replay items: connects and disconnect, parse execute and cancel. It should be simple to collect.

For the session statistics, just add one for every connect, subtract one for every disconnect and remember the maximum value.

It should be simple.